### PR TITLE
Add python implementation of nqueens using backtracking

### PR DESCRIPTION
**Pull request by nirajpandkar**

### CODE REVIEW

File: code/backtracking/n_queen/nqueen.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+import json
```
Comment (hwchiu) [inline code/backtracking/n_queen/nqueen.py]: It seems we don't need jsom library here, could you remove this unused imprort?

File: code/backtracking/n_queen/nqueen.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+global x
+x=[]
+global n
+n=0
+global flag
```
Comment (hwchiu) [inline code/backtracking/n_queen/nqueen.py]: how about we initialize all global variable together?
Maybe in here or in main function?

File: code/backtracking/n_queen/nqueen.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+				flag=1
+			else:
+				nqueens(k+1)
+
+
+def main():
+	global n
+	global x
+	global flag
+	flag=0
```
Comment (hwchiu) [inline code/backtracking/n_queen/nqueen.py]: As mentioned above, initialize all global variable together.

File: code/backtracking/n_queen/nqueen.py (hunk 1 above)
Comment (nirajpandkar) [reply code/backtracking/n_queen/nqueen.py]: Oops. Got past me. Will remove it.

File: code/backtracking/n_queen/nqueen.py (hunk 2 above)
Comment (nirajpandkar) [reply code/backtracking/n_queen/nqueen.py]: Sure thing.

**Comment by nirajpandkar** [comment]
Done :+1:

**Comment by hwchiu** [comment]
Hi @AdiChat  
This patch looks good to me.

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**